pages/common/ansiweather.md

```diff
@@ -0,0 +1,15 @@
+# ansiweather
+
+> AnsiWeather is a Shell script for displaying the current weather conditions in your terminal.
```

Comment:
Not sure we need `AnsiWeather is` here. Also, I don't think `Shell` should be capitalised.